build.sbt

```diff
@@ -27,7 +27,11 @@ lazy val projectSettings = Seq(
   dependencyOverrides ++= Seq(
     "io.kamon" %% "kamon-core" % kamonVersion
   ),
-  fork := true
+  fork := true,
```

Comment:
You can as well remove the fork. It's set globally in projectSettings.

Reply:
✅